### PR TITLE
2 create account login page

### PR DESCRIPTION
**Pull request by im-addie**
- created a login page for users to log in to their accounts
- routes users back to the homepage if their sign-in was successful
- shows an error if the user inputs the wrong credentials

### CODE REVIEW

File: ui/src/App.js
Comment (Amin-JRS) [inline ui/src/App.js:1]: make path '/Login' lowercase on line 18

File: ui/src/views/Login/index.js
Comment (Amin-JRS) [inline ui/src/views/Login/index.js:1]: The user isn't logged in unless you run `setToken(token)` with the result from the login api call. Add this before redirecting user. 

Line 99: change Link to `<Link to={'/forgot-password'}>`
Line 106 change Link to `<Link to={'/register'}>`

**Review by Amin-JRS — CHANGES_REQUESTED** [review]
Make changes before approval

**Comment by im-addie** [comment]
Requested changes have been resolved.

**Pull request merged**